scripts/analyst_collective/runner.py

```diff
@@ -12,12 +12,17 @@ def __init__(self, config, creds, models_dir):
     def models(self):
         return set(self.config['models'])
 
-    def try_create_schema(self):
+    def drop_schema(self):
+        sql = self.interpolate("drop schema if exists {schema} cascade")
+        self.execute(sql)
+
+    def create_schema(self):
         sql = self.interpolate("create schema if not exists {schema}")
```

Comment:
if we're always doing the drop, we should remove the `if not exists` from this call.  I'd rather see a hard error indicating that I wasn't working with a clean-slated schema than debug something related to a conflict with a dirty schema